jujusvg.go

```diff
@@ -13,8 +13,33 @@ import (
 // NewFromBundle returns a new Canvas that can be used
 // to generate a graphical representation of the given bundle
 // data. The iconURL function is used to generate a URL
-// that refers to an SVG for the supplied charm URL.
-func NewFromBundle(b *charm.BundleData, iconURL func(*charm.Reference) string) (*Canvas, error) {
+// that refers to an SVG for the supplied charm URL.  If
```

Comment:
I'd be tempted to phrase this slightly differently.

    // If fetcher is non-nil, it will be used to fetch icon
    // contents for any icons embedded within the charm,
    // allowing the generated bundle to be self-contained.